portal-survey-2023.Rmd

```diff
@@ -0,0 +1,291 @@
+---
+title: "NF-OSI Data Access Survey -- 2023 Preliminary Results"
+description: |
+  Preliminary survey findings from the NF community on the experience of data discovery, sharing, and reuse through the NF Data Portal.
+author:
+  - name: Anh Nguyet Vu
+    url: https://github.com/anngvu
+    affiliation: Sage Bionetworks
+    affiliation_url: https://sagebionetworks.org/
+    orcid_id: 0000-0003-1488-6730
+  - name: Jineta Banerjee
+    url: https://github.com/jaybee84
+    affiliation: Sage Bionetworks
+    affiliation_url: https://sagebionetworks.org/
+    orcid_id: 0000-0002-1775-3645
+  - name: Christina Conrad
+    url: https://github.com/cconrad8
+    affiliation: Sage Bionetworks
+    affiliation_url: https://sagebionetworks.org/
+    orcid_id: 0000-0001-8688-2523
+  - name: Robert Allaway
+    url: https://github.com/allaway
+    affiliation: Sage Bionetworks
+    affiliation_url: https://sagebionetworks.org/
+    orcid_id: 0000-0003-3573-3565
+date: "`r Sys.Date()`"
+output: distill::distill_article
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+library(gt)
+library(googlesheets4)
+library(tidygraph)
+library(ggraph)
+
+source("R/circlepack.R")
+```
+
+
+```{r, include=FALSE}
+sheet_id <- "1mZ6_XzC1d82cCl8uBqBjSDvIMo7-QN4Q2_FMIifbmkA"
+raw <- googlesheets4::read_sheet(ss = sheet_id)
+
+```
+
+```{r, echo=F}
+# users
+emails <- raw %>%
+  pull(`Email Address`) %>%
+  unique()
+
+deid <- setNames(1:length(emails), emails)
+survey <- raw %>%
+  mutate(id = deid[`Email Address`]) %>%
+  select(-c(`Email Address`))
+
+# use only most recent response where some users submitted multiple responses
+survey <- survey %>%
+   group_by(id)  %>%
+   top_n(n = 1, Timestamp)
+```
+
+## Summary
+
+The Neurofibromatosis (NF) community survey conducted to assess data findability, sharing, and reuse within the NF Data Portal/Platform has captured feedback from diverse users, with strong representation for Principal Investigators (PIs). Preliminary results indicate that a confusing user interface and poor search functionality were primary obstacles to data findability and affected the most users. Data sharing was particularly hampered by poor instructions and process flow, while data reuse's main issue was data download capacity. Future efforts include getting the perspectives of rare disease patients, as well as conducting more qualitative research. The ongoing survey aims to gather additional data to address limitations in the initial responses and track changes over time.
```

Comment:
A few observations here:
1. As written this reads very negative.
2. It would be great to highlight that the next steps would include specific actions towards improvements of the identified areas, and not just more qualitative surveys.

Reply:
Good points, I revised the summary